src/lib/polyfills/browser-polyfills.ts

```
@@ -0,0 +1,334 @@
+ * This file provides browser-friendly versions of Node.js modules
+ * that are being used in browser code.
+ */
+
+// Process polyfill
+export const process = {
+  env: {
+    NODE_ENV: import.meta.env.MODE || 'development',
+    // Add any other environment variables needed in browser context
+    BROWSER: 'true',
+  },
+  nextTick: (callback: Function, ...args: any[]) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)